app/code/Magento/Developer/Model/TemplateEngine/Plugin/DebugHints.php

```diff
@@ -58,24 +59,51 @@ class DebugHints
     protected $debugHintsPath;
 
     /**
+     * XPath of configuration of the debug hints show with parameter
+     *
+     *     dev/debug/template_hints_storefront_show_with_parameter
+     *
+     * @var string
+     */
+    protected $debugHintsShowWithParameter;
```

Comment:
Current Magento Technical Vision does not recommend designing classes for extensibility, therefore please declare properties as `private`.